### PR TITLE
 MSA aggregation

### PR DESCRIPTION
**Pull request by tiffanychu90**
* Make crosswalk of county FIPS to CBSA FIPS (combination of this [NBER crosswalk](https://data.nber.org/data/cbsa-fips-county-crosswalk.html) and [Alex's population table](https://docs.google.com/spreadsheets/u/0/d/16dbgSRn61WmhWtZbOnDvHCExiDVEK383tEMHnvAhPWQ/edit?usp=drive_web)), and upload to S3. Upload crosswalk to our GitHub too.
* Aggregate cases and deaths to MSA and calculate rate per 1M

Ref [GitHub issue](https://github.com/CityOfLosAngeles/aqueduct/issues/199)

### CODE REVIEW

**Comment by tiffanychu90** [comment]
@hunterowens start with the cbsa-county-crosswalk notebook. My latest commit removes the work in creating the crosswalk, and now it's in S3, but I think we should put the crosswalk into our GitHub too?

**Comment by hunterowens** [comment]
Here's a DAG version. @ian-r-rose @tiffanychu90

File: dags/public-health/covid19/jhu-county-to-esri.py [hunk 1]
```diff
@@ -264,6 +266,100 @@ def integrify(x):
...
+    Update MSA dataset
+    ref gh/aqueduct#199
+    takes the previous step data, aggegrates by MSA
+    replaces featurelayer.
+    """
+    # arcconnection = BaseHook.get_connection("arcgis")
+    # arcuser = arcconnection.login
+    # arcpassword = arcconnection.password
+    arcuser = "hunterowens"
+    arcpassword = "EDBQuESzo9j9"
```
Comment (ian-r-rose) [inline dags/public-health/covid19/jhu-county-to-esri.py:309]: Oops!

Comment (tiffanychu90) [reply dags/public-health/covid19/jhu-county-to-esri.py:309]: I took it out

**Comment by tiffanychu90** [comment]
@hunterowens Can you test the DAG again? I just noticed NYC isn't in the final result, but I think I restored it now?

**Comment by ian-r-rose** [comment]
You'll need to edit commit history, it's still there

File: dags/public-health/covid19/jhu-county-to-esri.py (hunk 1 above)
Comment (hunterowens) [reply dags/public-health/covid19/jhu-county-to-esri.py]: test now? I hopefully rebased succesfully?

Comment (hunterowens) [reply dags/public-health/covid19/jhu-county-to-esri.py]: ugh nope

**Pull request merged**